### ISSUE TITLE
about tizen webos project

### ISSUE THREAD

**Issue by rawgimaster** [smarttvdev/Test1#1]
i need you on tizen webos project,

can you please drop where i can contact you

**Comment by smarttvdev** [comment]
can u please let me know who you are?

**Comment by rawgimaster** [comment]
Sure, im trying to find a developer that will the job that you have did, and i found out that you are the developer that mad the app that i was trying to do,  Author name: BaiMaoLi
password: BaiMaoLi1

**Comment by rawgimaster** [comment]
![image](https://github.com/user-attachments/assets/54ef1b13-5746-4b74-b5ad-58d0dcf05878)
im talking about the ibo player for tizen webos

**Comment by smarttvdev** [comment]
Sorry, I am not making any new app any more.

**Comment by rawgimaster** [comment]
i understand, but im looking exaclty for the old app that you have made. the code source of the this app that you have made in 2023

**Comment by smarttvdev** [comment]
Are you a developer?
And need source code?
How did u find my contact?

**Comment by rawgimaster** [comment]
yes im a developer, i downloaded the userwidget tmg file, i tried to reverse it, and i end up to file the "BaiMaoLi1" in the tizen certificate name, i did a little research and i endup with this github user

**Comment by smarttvdev** [comment]
Great.
So, you could get my source files also?

**Comment by rawgimaster** [comment]
nop, i wasn't able to do that :(

**Comment by smarttvdev** [comment]
Hmm. I see.

**Comment by rawgimaster** [comment]
im android developer i have built smillar app like the app you developed, but im not able to write the full app for samasung tv tizen. so im stuck :(

**Comment by rawgimaster** [comment]
so could you please help me, i can pay for it if you want to!  :)

**Comment by smarttvdev** [comment]
Unfortunately, I can not help you with it.

**Comment by rawgimaster** [comment]
It’s available for sale ? :(

but you said you don’t do this anymore;(
Very sad

**Comment by rawgimaster** [comment]
Hello ?
